### DISCUSSION TITLE
エラーを吐いてしまい教科書の先に進めません！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#130] [Question-original] [answered]
## (1) 実現したいこと
メモを何個も保存できるアプリ作成
## (2) 実現するために自分が試した内容とその結果
教科書の45ページまでやりました
## (3) 調べた内容や資料
ColletionView2の教科書
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
![image](https://user-images.githubusercontent.com/75009903/217497655-aa58caa0-e329-4993-8ac7-5671403aeebc.png)
## (5) GitHubのリンク
https://github.com/KOTARO1209/MemoPad

**Comment by txxxxc** [comment, marked as answer]
Cellのidentifierが設定されていないor関連付けができてない気がするので、確認してみてほしい！
Discordで解決

**Reply by KOTARO1209** [reply]
![image](https://user-images.githubusercontent.com/75009903/217835306-9b619f37-df7a-430d-85d1-a3b3db0e4a1b.png)
![image](https://user-images.githubusercontent.com/75009903/217835407-4990e2f3-92c8-4e5b-b668-11a5dd6041fd.png)

identifierは書いてあって、関連付けにCellが見つからないのですがどこでしょうか🙇‍♂️

**Reply by txxxxc** [reply]
ここって正しく設定できてる？
![image](https://user-images.githubusercontent.com/22112440/217839107-dd4dec78-6901-417b-a3a0-b2b412b7a6c1.png)

**Reply by KOTARO1209** [reply]
![image](https://user-images.githubusercontent.com/75009903/217839821-81fa1f2f-07f8-4554-bee7-8f6c73ebf6d2.png)
たぶんできてます
